src/App.test.js

```diff
@@ -1,9 +1,90 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+const puppeteer = require('puppeteer');
+const faker = require('faker');
+
+// Uses faker to set inital "person" object, This can be used whenever you need to fill in a form with any param inside the object.
+const person = {
+  name: `${faker.name.firstName()} ${faker.name.lastName()}`,
+  email: "c@lvin.com",
+  phone: faker.phone.phoneNumber(),
+  message: faker.random.words(),
+  password: "password"
+};
+
+
+// Inital setup for pupeteer
+let browser;
+let page;
+const width = 1920;
+const height = 1080;
+
+// Setup before any test is run, launches puppeteer with the window sizes & heigh
+beforeAll(async () => {
+  browser = await puppeteer.launch({
+    headless: true,
+    slowMo: 80,
```

Comment:
Remove `slowMo` (just to make it a bit faster when testing), and add a `page.wait()` at necessary parts (e.g. when an API request is made, when something new needs to load, etc.) If this turns out to be unwieldy, we can revert this later.